qa/tasks/cephfs/test_dir_charmap.py

```diff
@@ -342,6 +342,20 @@ def test_cs_not_empty_set_insensitive(self):
             else:
                 self.fail("should fail")
 
+    def test_charmap_to_stray(self):
+        """
+        That internal renames for reintegration works with charmap.
+        """
+
+        self.mount_a.run_shell_payload("mkdir foo/")
+        self.mount_a.setfattr("foo/", "ceph.dir.casesensitive", "0")
```

Comment:
this should fail even if it's set to `1`, maybe add a comment stating "toggling any value works, we just want to have a charmap attribute set on the dir" or something?

Reply:
ACK